docs/pyspark.md

```diff
@@ -5,7 +5,7 @@ Make sure python is installed on your system. Then, execute the following comman
 that ships with Apache Spark.
 
 ```bash
-$ spark-client.pyspark
+$ spark-client.pyspark --conf spark.driver.host={LOCALHOST_IP}
```

Comment:
This localhost ip business, should we take care of it like we do autodetection of k8s master url?

Reply:
yes, that a good point, but let's leave it as you suggest as next improvement